scripts/check-sample-lnd-conf.sh

```diff
@@ -14,33 +14,35 @@ CONF_FILE=${2:-sample-lnd.conf}
 # We are reading the default values of lnd from lnd --help. To avoid formatting
 # issues the width of the terminal is set to 240. This needs a workaround for
 # CI where we don't have an interactive terminal.
+FILE_TMP=$(mktemp)
 if [ -t 0 ]; then
     stty cols 240
-    LND_HELP="$(go run -tags="$TAGS" \
-        github.com/lightningnetwork/lnd/cmd/lnd --help)"" --end"
+    go run -tags="$TAGS" github.com/lightningnetwork/lnd/cmd/lnd --help > \
+        $FILE_TMP
 else
     tmux new-session -d -s simulated-terminal -x 240 -y 9999
     tmux send-keys -t simulated-terminal.0 "go run -tags=\"$TAGS\" \
-        github.com/lightningnetwork/lnd/cmd/lnd --help; tmux wait -S run" ENTER
+        github.com/lightningnetwork/lnd/cmd/lnd --help >"$FILE_TMP"; \
+        tmux wait -S run" ENTER
     tmux wait-for run
-    LND_HELP="$(tmux capture-pane -t simulated-terminal.0 -p)"" --end"
     tmux kill-session -t simulated-terminal
 fi
 
+LND_HELP="$(cat $FILE_TMP)"" --end"
```

Comment:
Do we need those two double quotes in the middle? Looks like it should work without them as well.

Reply:
not really. personal habit.